onnxruntime/python/tools/quantization/onnx_model.py

```diff
@@ -59,6 +62,14 @@ def remove_initializers(self, init_to_remove):
         for initializer in init_to_remove:
             self.remove_initializer(initializer)
 
+    def get_non_initializer_inputs(self):
+        initializer_names = self.get_initializer_name_set()
+        non_initializer_inputs = set()
```

Comment:
If set is used, later usage for generated_value_names better also change to set() semantic

Reply:
get_non_initializer_inputs returns set type and the generated_value_names is assigned with the set

Reply:
I mean later code is using it as map